src/platform/web/transport/transport-options.ts

```diff
@@ -31,4 +31,6 @@ export interface TransportOptions {
    * @defaultValue `true`
    */
   traceSip?: boolean;
+
+  transportProtocols?: string[]
```

Comment:
@kkbhanuchandar please add ; at the end
also lets add some descriptive comment like in the above so we can also contribute it back.

Reply:
resolved.